source-code/sdk-js/src/detectors/matchLang.ts

```diff
@@ -0,0 +1,17 @@
+import type { Language } from "./sharedTypes.js"
+
+export const matchLang = (detected: Language[], availableLanguages: Set<Language>) => {
```

Comment:
please use good variable names and don't short words where we need to think what this could mean